config.toml

```diff
@@ -3,4 +3,4 @@ languageCode = "en-us"
 title = "ALC Beijing"
 disableKinds = ["taxonomy", "taxonomyTerm"]
 theme = "tale"
-canonifyURLs = true
```

Comment:
But why remove this, it breaks the build

Reply:
To fix the image display issue, we need to set uglyurls = true.

Reply:
> To fix the image display issue, we need to set uglyurls = true.

Can't `canonifyURLs` and `uglyurls` exist simultaneously?

Reply:
I just find another to workaround the relative path issue.